packages/trigger-sdk/src/triggers/dynamic.ts

```diff
@@ -81,14 +81,18 @@ export class DynamicTrigger<
         payload: deepMergeFilters(this.source.filter(params), this.event.filter ?? {}),
       },
       source: {
+        version: "2",
         key,
         channel: this.source.channel,
         params,
-        events: typeof this.event.name === "string" ? [this.event.name] : this.event.name,
+        //todo add other options here
+        options: {
+          event: typeof this.event.name === "string" ? [this.event.name] : this.event.name,
+        },
```

Comment:
@ericallam currently this just has the event name, I couldn't figure out how we would support other options here

Reply:
I think that's fine for now, we can sort out the other options later if needed.